### DISCUSSION TITLE
Newbie need help!!!

### DISCUSSION THREAD

**Discussion by audiophilone** [UI-Lovelace-Minimalist/UI#1446] [Q&A]
Hey...iam a newbie in HA...what is my fault?
![Homeassistant fault](https://github.com/UI-Lovelace-Minimalist/UI/assets/154750952/52e3b1a0-0456-45d0-a7b3-ad93514b56f9)

greetings from Germany!

**Comment by Ruu-D** [comment]
Probably indention error in YAML structure, or you have made a typo. Can you show us the file "home.yaml"?

**Reply by audiophilone** [reply]
thx for your answer...here is my file
![yaml file](https://github.com/UI-Lovelace-Minimalist/UI/assets/154750952/0a4ad1f4-21e3-403b-90ed-08327e041c2c)

**Reply by Ruu-D** [reply]
Use another 'TAB' indention before "template". Must match underneath 'type'.

**Reply by audiophilone** [reply]
now there is nothing in the dashboard
![yaml file2](https://github.com/UI-Lovelace-Minimalist/UI/assets/154750952/929427db-285b-49bb-ab96-53cbdcf3583f)

**Comment by audiophilone** [comment]
i need help :(
